### PR TITLE
fix(ui): user-related nav links to the right

### LINKED ISSUE #6: Align user login and registration nav items to the right
Items on the navbar pertaining to user login or registration should be aligned to the right of the navbar

### PR DESCRIPTION
**Pull request by zautumnz**
Closes #6

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -2,4 +2,9 @@
   margin-top: 5%;
   margin-left: 10%;
   margin-right: 10%;
-}
\ No newline at end of file
+}
+
+#navbar-wrapper {
```
Comment (zautumnz) [inline static/css/styles.css:7]: Here is the CSS change for the nav

File: templates/main.html [hunk 2]
```diff
@@ -18,22 +18,24 @@
         </button>
 
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
-            <ul class="navbar-nav mr-auto">
+            <ul id="navbar-wrapper" class="navbar-nav">
```
Comment (zautumnz) [inline templates/main.html:21]: Added an id here to target

File: templates/main.html [hunk 3]
```diff
@@ -18,22 +18,24 @@
...
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" href="/login">Log In</a>
-                </li>
-                {% else %}
-                <li class="nav-item">
-                    <a class="nav-link" href="/logout">Log Out</a>
-                </li>
-                {% endif %}
+                <ul class="navbar-nav">
```
Comment (zautumnz) [inline templates/main.html:25]: Made this a nested ul so this change could use easy flexbox styles rather than messing with margins and junk

**Review by aeshthetic — APPROVED** [review]
This looks good; I appreciate the fix a lot!

**Pull request merged**